instance/static/html/instance/index.html

```diff
@@ -10,7 +10,7 @@
   <!-- Instances List -->
   <div class="large-3 columns">
     <ul class="side-nav side-instance-list">
-      <li ng-repeat="instance in instanceList | filter: {is_shut_down: false}"
+      <li ng-repeat="instance in instanceList | filter: {is_archived: false}"
```

Comment:
I have changed the API to return only non-archived instances by default, but to still allow listing archived instances if desired.